README.md

```diff
@@ -1,2 +1,148 @@
-# Portfolio-setup-and-mobile-first
-Portfolio project
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>README</b></h3>
+</div>
+
+<br>
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [🚀 Live Demo](#live-demo)
```

Comment:
- [ ] Since you do not have a live demo, you can kindly remove this link to it :+1: